authui/src/authgear.ts

```diff
@@ -253,3 +264,51 @@ window.api.onLoad(() => {
     }
   };
 });
+
+window.api.onLoad(() => {
```

Comment:
`api.onLoad` handler would be called repeatedly whenever navigation happens, so WebSocket would be connected & disconnected repeatedly. I think we should subscribe to window's load event instead?